src/eventgrid/azext_eventgrid/_help.py

```diff
@@ -32,7 +32,7 @@
   - name: Create a new domain that accepts events published in CloudEvents V1.0 schema and maps a property mytopicfield to the topic name.
     text: az eventgrid domain create -g rg1 --name domain1 -l westus2 --input-schema cloudeventschemav1_0 --input-mapping-fields topic=mytopicfield
   - name: Create a new domain which allows specific inbound ip rules.
-    text: az eventgrid domain create -g rg1 --name domain1 -l westus2 --allow-traffic-from-all-ips false --inbound-ip-rules 10.0.0.0/8 Allow --inbound-ip-rules 10.2.0.0/8 Allow
```

Comment:
removing params is breaking change, is the cmd preview?

Reply:
The parameter is changed based on the recommendation from NRP. This parameter/feature is still in preview.

Reply:
for preview command, pls add preview tag on the command. HI @fengzhou-msft could you pls share an example? thanks.

Reply:
You can add `is_preview=True` for [command](https://github.com/Azure/azure-cli-extensions/blob/15231d881e14322c4f8566b4ea87509c608bcb83/src/front-door/azext_front_door/commands.py#L110) or [command group](https://github.com/Azure/azure-cli-extensions/blob/d7af04f437aa25b9d0aa2d260cf3bd050f76fe8b/src/azure-firewall/azext_firewall/commands.py#L73).

Reply:
We plan to use the mainstream cli for adding preview tags.. this is separate effort which requires moving all these preview commands/parameters to regular mainstream cli. For now, we will go with the current approach where cli extension is used for preview.